src/test/resources/ci/scripts/run.sh

```diff
@@ -17,6 +17,37 @@ currentTime=(date +"%Y/%m/%d-%H:%M:%S:%3N")
 # Operating system.
 OS=$(uname -s)
 
+# Primes the environment before running the tests.
+# This decreases the possibility of intermittent test issues
+# related to varying dependency download times.
```

Comment:
I think these changes are there to stay because the first test is what gets affected by dependency downloads. The same would be true after the mentioned issue is fixed. Since the issue should improve further performance, I expanded the comment a bit to make it more clear and mention the issue.